src/db.rs

```diff
@@ -144,7 +144,7 @@ impl DB {
                             continue;
                         }
                     };
-                    let path_abs = match Path::new(path_str).canonicalize() {
+                    let path_abs = match dunce::canonicalize(Path::new(path_str)) {
```

Comment:
The `str` type implements `AsRef<Path>` (which `dunce::canonicalize` uses as an argument), so `Path::new` is unnecessary here.